proposals/collection-expression-arguments.md

```diff
@@ -268,3 +268,84 @@ strongly triggers the view that this is simply an implicit-object-creation.  And
 case where a constructor *is* actually called (like for `Dictionary<,>`) it is misleading when calling a *create
 method*, or creating an interface.  Finally, there is general apprehension around using `new` at all as there
 is a feeling of redundancy around both the `new` indicating a new instance, *and* `[...]` indicating a new instance.
+
+## Conversions
+
+Collection arguments are *not* considered when determining *collection expression* conversions.
+
+## Construction
+
+Construction is updated as follows.
+
+The elements of a collection expression are evaluated in order, left to right.
+Within *collection arguments*, the arguments are evaluated in order, left to right.
+Each element or argument is evaluated exactly once, and any further references refer to the results of this initial evaluation.
+
+If *collection_arguments* is not the first element in the collection expression, a compile-time error is reported.
```

Comment:
Could be read to mean "if the first element is not a *collection_arguments* element" which would imply an error when there is no *collection_arguments* element.

```suggestion
If *collection_arguments* appears in the collection expression and is not the first element, a compile-time error is reported.
```